modules/identities/scripts/get.ts

```diff
@@ -0,0 +1,58 @@
+import { ScriptContext } from "../module.gen.ts";
+import { IdentityData, IdentityProviderInfo } from "../utils/types.ts";
+
+export interface Request {
+	userToken: string;
+    info: IdentityProviderInfo;
+}
+
+export interface Response {
+    data: {
+        uniqueData: IdentityData;
+        additionalData: IdentityData;
+    } | null;
+}
+
+export async function run(
+	ctx: ScriptContext,
+	req: Request,
+): Promise<Response> {
+	await ctx.modules.rateLimit.throttle({
```

Comment:
this is internal, should not be throttled, right? unless external providers depend on this throttle. if so, document that.

Reply:
They should not— will remove.